package.json

```diff
@@ -91,6 +91,7 @@
     "@loadable/webpack-plugin": "^5.7.1",
     "compression": "^1.7.4",
     "core-js": "^3.4.5",
+    "cross-env": "^6.0.3",
```

Comment:
> cross-env should be a regular dependency, rather than a dev. dependency, because it's used by scripts like npm start which will be run in prod, that does not install dev. dependencies

https://github.com/bbc/simorgh/issues/4667#issuecomment-557599303